molgenis-data-mapper/src/main/java/org/molgenis/data/mapper/controller/MappingServiceController.java

```diff
@@ -120,8 +137,6 @@ public String addMappingProject(@RequestParam("mapping-project-name") String nam
 			@RequestParam("target-entity") String targetEntity)
 	{
 		MappingProject newMappingProject = mappingService.addMappingProject(name, getCurrentUser(), targetEntity);
-		// FIXME need to write complete URL else it will use /plugin as root and the molgenis header and footer wont be
-		// loaded
 		return "redirect:/menu/main/mappingservice/mappingproject/" + newMappingProject.getIdentifier();
```

Comment:
The redirect is there as part of a https://en.wikipedia.org/wiki/Post/Redirect/Get to prevent double submission of the post. If you forward, the client doesn't see the GET and upon reload will do another POST.